Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine:3.11 as base
+
+RUN apk add curl
+
+RUN apk add python3
+
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python3 -
+
+COPY . .
+
+RUN source $HOME/.poetry/env && poetry update && poetry install
+
+FROM base as production
+ENV FLASK_ENV=production
+ENTRYPOINT ["gunicorn", "app:app"]
```

Comment:
Not strictly part of this module, but worth noting that this currently doesn't work - gunicorn is installed under poetry so it will need to be run with `poetry run gunicorn ..."